components/fieldgroups/experience-event/experienceevent-content.schema.json

```
@@ -0,0 +1,71 @@
+{
+  "meta:license": [
+    "Copyright 2023 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-content",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Content Information",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Content analytics data such as eventType, experience ID, and assets IDs.",
+  "definitions": {
+    "experienceevent-content": {
+      "properties": {
+        "xdm:experienceContent": {
+          "title": "Experience content",
+          "type": "object",
+          "description": "Experience information content.",
+          "properties": {
+            "xdm:contentEventType": {
+              "title": "Content Event Type",
+              "type": "string",
+              "meta:enum": {
+                "view": "View",
+                "click": "Click"
+              },
+              "description": "The primary content event type."
+            },
+            "xdm:experience": {
+              "title": "Content Experience",
+              "description": "Content experience tracking",
+              "type": "object",
+              "properties": {
+                "xdm:experienceID": {
+                  "title": "Content Experience ID",
+                  "type": "string",
+                  "description": "Content experience identifier."
+                }
+              }
+            },
+            "xdm:assets": {
```

@ilievlad73 - one more thing:
The `xdm:assets` node has only one property, which is `xdm:assetsIDs`. Do you plan to have additional properties in `xdm:assets` nodes in the future? If yes, then following is the recommended structure:
```suggestion
    "xdm:assets": {
        "title": "Content Assets",
        "description": "Content assets tracking",
        "type": "array",
        "items": {
            "title": "Asset Properties",
            "description": "Additional properties of the assets",
            "type": "object",
            "properties": {
                "xdm:assetID": {
                    "title": "Content Assets ID",
                    "description": "Content assets identifier.",
                    "type": "string"
                }
            }
        }
    }
```    
That way in the structure above, you can add additional properties such as `xdm:assetName` next to `xdm:assetID` property, and keep a room for extending it further in the future.

If you don't plan to have any additional properties, then you should eliminate the `xdm:assets` node and bubble-up the existing `"xdm:assetIDs` node in its place.

---

I will bubble-up the existing "xdm:assetIDs node.